website/.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
we shouldn't duplicate the top folder configuration that applies to all the files

Reply:
By default prettier will also search in parent folder for a configuration

Reply:
there was some issue were something was looking for `website/website`

Reply:
I think it's more for eslint but we should still not duplicate linter/formatter rules across the projet